galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager/src/main/java/dev/galasa/core/manager/internal/gherkin/GherkinLog.java

```diff
@@ -0,0 +1,26 @@
+package dev.galasa.core.manager.internal.gherkin;
+
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.logging.Log;
+
+public class GherkinLog {
+
+    private final static Pattern patternVariable = Pattern.compile(".*(<\\w+>).*");
```

Comment:
dont code .*  not good in regex,    "(<\\w+>)" is enough,  as that is what you are looking for

Reply:
Best if you actually do
"(<(\w+)>)"

then the variable name is in group 2